app/views/vaccinate/warning-wrong-age-for-product.html

```diff
@@ -0,0 +1,31 @@
+
+{% extends 'layout.html' %}
+
+{% set pageName = "The vaccine product may not be recommended for someone this age" %}
+
+{% set currentSection = "vaccinate" %}
+
+{% block beforeContent %}
+  {{ backLink({ href: "/vaccinate/patient-history" }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1 class="nhsuk-heading-l">Adjuvanted Quadrivalent Influenza Vaccine (aQIV) may not be recommended</h1>
+
+      <p>
+        {{ data.patientName }} is 64 years old. Adjuvanted Quadrivalent Influenza Vaccine (aQIV) is not usually recommended for people under the age of 65.</p> 
+        <p>Check before going ahead or refer to a prescriber for a Patient Specific Direction.</p>
+      
+          </div>
+  </div>
```

Comment:
This is a nitpick, sorry, but can be helpful to try and line up the indentation.

```suggestion
      <p>
        {{ data.patientName }} is 64 years old. Adjuvanted Quadrivalent Influenza Vaccine (aQIV) is not usually recommended for people under the age of 65.</p> 
      <p>Check before going ahead or refer to a prescriber for a Patient Specific Direction.</p>
      
    </div>
  </div>
```

Reply:
Fair comment. Will take a look.